applications/asset_tracker_v2/prj.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_REBOOT=y
 CONFIG_FPU=y
 
 # Heap and stacks
-CONFIG_HEAP_MEM_POOL_SIZE=32768
+CONFIG_HEAP_MEM_POOL_SIZE=47250
```

Comment:
This heap size is now used for both the DK and Thingy91 which I'm pretty sure would cause SRAM overflow for some configurations of the application. For instance when building for lwm2m + (either modem trace or memfault).
i.e:

`west build -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG="overlay-debug.conf;overlay-lwm2m.conf;overlay-memfault.conf"`

or

adding `CONFIG_NRF_MODEM_LIB_TRACE=y` and running `west build -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG="overlay-debug.conf;overlay-lwm2m.conf"`

We should probably have two heapsizes that depends on the board that is used, they can be placed into an overlay file in the `app/boards/` folder.
The heap size for the DK can even possibly be reduced because it doesn't include encoding of environmentals and impact.

Currently the thingy91 uses SPM that has much less overhead compared to TFM that the DK uses. This is why they must differ.